docs/platforms/javascript/common/best-practices/multiple-sentry-instances.mdx

```diff
@@ -15,17 +15,26 @@ notSupported:
   - javascript.nestjs
 keywords:
   [
-    "multiple instances",
     "multiple clients",
     "BrowserClient",
     "NodeClient",
+    "Browser Extension",
+    "VSCode Extension",
+    "Widgets",
     "monorepo"
   ]
 ---
 
+These best practices are relevant for you if you set up Sentry in one of the following use-cases:
+- Browser Extensions
+- VSCode Extensions
+- Third-Party Widgets
+- Libraries
+- Any other scenario where you might have multiple Sentry instances running in the same environment
+
 <Note>
 
-Creating multiple Sentry clients is not recommended in general, as it can lead to unexpected behavior. If you are using Micro Frontends or similar, multiplexing might be a better solution that using multiple clients. Check out [Micro Frontends](/platforms/javascript/best-practices/micro-frontends) in the Best Practices for more information.
+Creating multiple Sentry clients is **not recommended** in general, as it can lead to unexpected behavior. In case you are using Micro Frontends or similar, multiplexing might be a better solution that using multiple clients. Check out [Micro Frontends](/platforms/javascript/best-practices/micro-frontends) in the Best Practices for more information.
```

Comment:
```suggestion
Creating multiple Sentry clients is **not recommended** in general, as it can lead to unexpected behavior. In case you are using Micro Frontends or similar, multiplexing might be a better solution than using multiple clients. Check out [Micro Frontends](/platforms/javascript/best-practices/micro-frontends) in the Best Practices for more information.
```